exercise/api/csv/views.py

```diff
@@ -353,6 +353,102 @@ def get_renderer_context(self):
         context['header'] = getattr(self, 'renderer_fields', None)
         return context
 
+class CourseResultsDataViewSetBest(NestedViewSetMixin,
```

Comment:
I know we discussed that we should start with a quickly deployable (even if ugly) fix to make the results page work again ASAP, but we should nevertheless try to avoid unnecessary code duplication, when it is easily doable: there is only line that differs from the other function as far as I can tell. For example, no need to copy-paste comments word-to-word. Instead the description text could tell that this is a quick fix that will be improved later. If this class would inherit the original, at least the completely duplicate definitions could be avoided, right? Also, if inheritance relationship was applied, splitting the serialize_profiles function into common parts and to the differing part that collects 'aggr' would perhaps be another way of avoiding completely duplicate code? (e.g. a 'aggr = collect_submissions' function that will be separately implemented in the two classes, otherwise the serialize_profiles implementation seems common)